src/javascript/annotate.ts

```diff
@@ -0,0 +1,14 @@
+import {isPathImport} from "../path.js";
+
+/**
+ * Annotate a path to a local import or file so it can be reworked server-side.
+ */
+
+const annotate = process.env["OBSERVABLE_ANNOTATE_FILES"];
+if (typeof annotate === "string" && annotate !== "true")
+  throw new Error(`unsupported OBSERVABLE_ANNOTATE_FILES value: ${annotate}`);
+export default annotate
```

Comment:
We don’t use default exports (because there’s no real reason to, and it encourages more inconsistency in how people name an imported symbol). This should be called `annotatePath` or something.